sycl/test-e2e/Graph/Inputs/stream.cpp

```diff
@@ -0,0 +1,56 @@
+// This test checks that we can use a stream when explicitly adding a
+// command_graph node.
+
+#include "../graph_common.hpp"
+
+int main() {
+  queue Queue;
+
+  using T = int;
+
+  size_t WorkItems = 16;
+  std::vector<T> DataIn(WorkItems);
+
+  std::iota(DataIn.begin(), DataIn.end(), 1);
+
+  exp_ext::command_graph Graph{Queue.get_context(), Queue.get_device()};
+
+  T *PtrIn = malloc_device<T>(WorkItems, Queue);
+  Queue.copy(DataIn.data(), PtrIn, WorkItems);
+
+  add_node(Graph, Queue, [&](handler &CGH) {
+    sycl::stream Out(WorkItems * 16, 16, CGH);
+    CGH.parallel_for(range<1>(WorkItems), [=](item<1> id) {
+      Out << "Val: " << PtrIn[id.get_linear_id()] << sycl::endl;
+    });
+  });
+
+  auto GraphExec = Graph.finalize();
+
+  Queue.submit([&](handler &CGH) { CGH.ext_oneapi_graph(GraphExec); });
+
+  Queue.wait_and_throw();
+
+  Queue.copy(PtrIn, DataIn.data(), Size);
+
+  free(PtrIn, Queue);
+
+  return 0;
+}
+
+// CHECK-DAG: Val: 1
+// CHECK-DAG: Val: 2
+// CHECK-DAG: Val: 3
+// CHECK-DAG: Val: 4
+// CHECK-DAG: Val: 5
+// CHECK-DAG: Val: 6
+// CHECK-DAG: Val: 7
+// CHECK-DAG: Val: 8
+// CHECK-DAG: Val: 9
+// CHECK-DAG: Val: 10
+// CHECK-DAG: Val: 11
+// CHECK-DAG: Val: 12
+// CHECK-DAG: Val: 13
+// CHECK-DAG: Val: 14
+// CHECK-DAG: Val: 15
+// CHECK-DAG: Val: 16
```

Comment:
To be consistent with the memory "LEAK" check, it is may be better to keep all llvm-lit checks in the test file which includes the run command. (?)